media-sound/loudgain/loudgain-0.6.8.ebuild

```diff
@@ -0,0 +1,26 @@
+# Copyright 2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+if [[ ${PV} == "9999" ]] ; then
+	EGIT_REPO_URI="https://github.com/Moonbase59/${PN}.git"
+	inherit git-r3
```

Comment:
Shouldn't be ```inherit``` before EGIT_*?

Reply:
I don't think the order matters. But I can change it if you prefer.

Reply:
Seems about random which line comes first, in current ebuilds.